crates/air-lib/interpreter-data/src/lib.rs

```diff
@@ -42,6 +42,8 @@ pub use trace_pos::*;
 
 use once_cell::sync::Lazy;
 use serde_json::Value as JValue;
+#[macro_use]
```

Comment:
These lines are not needed in modern Rust.

Reply:
Doesn't work w/o these two with:
root@2e598455657b /git/fluence-aquavm (chore/vm-267-trace-pos)$rustc --version
rustc 1.67.0-nightly (95a3a7277 2022-10-31)